assets/scripts/embedding_tests_dev_0.0.1.min.html

```diff
@@ -39,3 +45,20 @@ <h3>ERROR: 404 error</h3>
   <h3>ERROR: 404 error with deferred loading</h3>
   <oppia oppia-id="fake_id" autoload="false"></oppia>
 </div>
+
+<script type="text/javascript">
+  var currentExplorationId = 'idToBeReplaced';
```

Comment:
var EXPLORATION_ID_SENTINEL

(it's not really a "current exp ID", since it's not an exp ID at all...)

Reply:
I have change the name to 'placeHolderExplorationId'.